ci/run_testsuite_and_record.sh

```diff
@@ -6,12 +6,13 @@ set -e
 
 # clean up on exit, even if something fails
 function cleanup {
-    docker compose down
+    docker compose --ansi=never down
 }
 trap cleanup EXIT
 
 # start mreg+postgres in containers
-docker compose up -d
+docker compose --ansi=never pull --quiet
```

Comment:
For some reason, this fails for me with a local image (using `image: mreg/local:latest` in docker-compose.yml). I get `Error response from daemon: pull access denied for mreg/local, repository does not exist or may require 'docker login': denied: requested access to the resource is denied` when using pull and local images... Meh.

For now I simply comment out the pull command when using local images. Ideally we'd also make it easier to do use local images without editing the docker-compose file.

Reply:
I've modified `run_testsuite_and_record.sh` so it doesn't run `docker compose pull` if run locally, only when run from a github workflow. That should help

Reply:
It does indeed help! 👍